sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -6,7 +6,6 @@
     <template #actions>
       <v-row class="m-0 p-0">
         <v-text-field
-          v-if="!!offeringsAndCount.count"
```

Comment:
why did we remove the v-if here? is that a requirement to have search text irrespective of data returned?

Reply:
we have an alignment issue in some data tables (empty table). this condition is actually not needed in the search box.